app/services/find/match_old_params.rb

```diff
@@ -0,0 +1,62 @@
+module Find
+  class MatchOldParams
+    include ServicePattern
+
+    FILTERS = {
+      "senCourses" => "send_courses",
+      "lat" => "latitude",
+      "lng" => "longitude",
+      "rad" => "radius",
+      "query" => "provider.provider_name",
+      "hasvacancies" => "has_vacancies",
+      "subject_codes" => "subjects",
+    }.freeze
+
+    STUDY_FILTERS = {
+      "parttime" => "part_time",
+      "fulltime" => "full_time",
+    }.freeze
+
+    QAULIFICATION_FILTERS = {
+      "Other" => "pgce pgde",
+      "PgdePgceWithQts" => "pgce_with_qts",
+      "QtsOnly" => "qts",
+    }.freeze
+
+    def initialize(request_params)
+      @request_params = request_params
+    end
+
+    def call
```

Comment:
Can we break this method to smaller private methods, akin to the way you have tested it

@darokel I think we need to revisit https://github.com/DFE-Digital/publish-teacher-training/blob/main/config/rubocop/metrics.yml

It seems we induced needless tech debt.

Reply:
Done